collectors/UcsServerLicenseCollector.py

```diff
@@ -3,12 +3,13 @@
 
 
 class UcsServerLicenseCollector(BaseCollector):
-    def __init__(self, creds, inventory_file):
-        super().__init__(creds, inventory_file)
+    def __init__(self, creds, config):
+        super().__init__(creds, config)
         self.license_state = {
             "license-expired": 0,
-            "not-applicable": 1,
-            "license-ok": 2
+            "license-graceperiod": 1,
+            "not-applicable": 2,
+            "license-ok": 3
```

Comment:
Make sure you add this information to the readme. Always do it for collectors that use some self-defined status.

Reply:
Added in readme.